readme.md

```diff
@@ -149,6 +156,15 @@ case class Maybe2(@dropDefault i: Option[Int] = None)
 FromScala(Maybe2(None)).transform(ToJson.string)     ==> """{}"""
 ```
 
+But `Option` types are a special case where `None` is an assumed default if a default is not provided explicitly.
```

Comment:
could you add a test case proving this?

Reply:
https://github.com/rallyhealth/weePickle/blob/63e5b5648b8b3a0bbcaa4a5a389f9aecbc13f504/weepickle/test/src/com/rallyhealth/weepickle/v1/StructTests.scala#L149